webapp/store/models.py

```diff
@@ -5,9 +5,31 @@
 import gfm
 from jujubundlelib import references
 from theblues.charmstore import CharmStore
+from theblues.identity_manager import IdentityManager
 from theblues.errors import EntityNotFound
 
-cs = CharmStore("https://api.jujucharms.com/v5")
+cs = CharmStore('https://api.jujucharms.com/v5')
+
+
+
```

Comment:
we use black to auto lint python code 

(if you have locally: `black --line-length 79 webapp` )

Reply:
Thanks!